pp/integrations/plugin-packs/procedures/applications-biztalk.md

```diff
@@ -2,36 +2,297 @@
+| MSSQLUSERNAME | User name used to connect to the database |                   |             |
+| MSSQLPASSWORD | Password for the defined user name        |                   |             |
+| MSSQLPORT     | Database Server Port                      | 1433              |             |
+
+5. [Deploy the configuration](/docs/monitoring/monitoring-servers/deploying-a-configuration). The host appears in the list of hosts, and on the **Resources Status** page. The command that is sent by the connector is displayed in the details panel of the host: it shows the values of the macros.
+
+### Using a service template provided by the connector
+
+1. If you have used a host template and checked **Create Services linked to the Template too**, the services linked to the template have been created automatically, using the corresponding service templates. Otherwise, [create manually the services you want](/docs/monitoring/basic-objects/services) and apply a service template to them.
+2. Fill in the macros you want (e.g. to change the thresholds for the alerts). Some macros are mandatory (see the table below).
+
+<Tabs groupId="sync">
+<TabItem value="Received-Locations-Disabled" label="Received-Locations-Disabled">
+
+| Macro        | Description                                                                                        | Default value     | Mandatory   |
+|:-------------|:---------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| WARNING      | Warning threshold                                                                                  |                   |             |
+| CRITICAL     | Critical threshold                                                                                 |                   |             |
+| EXTRAOPTIONS | Any extra option you may want to add to the command (a --verbose flag for example). All options are listed [here](#available-options). | --verbose         |             |
+
+</TabItem>
+</Tabs>
+
+3. [Deploy the configuration](/docs/monitoring/monitoring-servers/deploying-a-configuration). The service appears in the list of services, and on the **Resources Status** page. The command that is sent by the connector is displayed in the details panel of the service: it shows the values of the macros.
+
+## How to check in the CLI that the configuration is OK and what are the main options for?
+
+Once the plugin is installed, log into your Centreon poller's CLI using the
+**centreon-engine** user account (`su - centreon-engine`). Test that the connector 
+is able to monitor a resource using a command like this one (replace the sample values by yours):
+
+```bash
+/usr/lib/centreon/plugins/centreon_biztalk.pl \
+	--plugin=database::mssql::plugin \
+	--dyn-mode=apps::biztalk::sql::mode::rlocationdisabled \
+	--hostname=10.0.0.1 \
+	--port=1433 \
+	--username='' \
+	--password='' \
```

Comment:
username and password

Reply:
Ne sont pas obligatoires pour lancer le plugin : 
```
perl centreon_plugins.pl --plugin=database::mssql::plugin --dyn-mode=apps::biztalk::sql::mode::rlocationdisabled --hostname=10.0.0.1
UNKNOWN: Cannot connect: OpenClient message: LAYER = (0) ORIGIN = (0) SEVERITY = (78) NUMBER = (44)
Server , database 
Message String: Server name not found in configuration files.
OpenClient message: LAYER = (0) ORIGIN = (0) SEVERITY = (78) NUMBER = (45)
Server , database 
Message String: Unknown host machine name.
OpenClient message: LAYER = (0) ORIGIN = (0) SEVERITY = (78) NUMBER = (41)
Server , database 
Message String: Unable to connect: Adaptive Server is unavailable or does not exist 
```